the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
```

Comment:
mysql_db.number7
DB Connection should be closed after use. Best practice is to have the connection handled in a helper class. This class will also execute queries, etc.